auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue

```diff
@@ -225,18 +224,14 @@ export default defineComponent({
       return NrState.DRAFT === state.toUpperCase()
     }
 
-    const isIA = (type: string): boolean => {
-      return (type === AffiliationTypes.INCORPORATION_APPLICATION || type === AffiliationTypes.REGISTRATION)
-    }
-
     const isProcessing = (state: string): boolean => {
       return NrDisplayStates.PROCESSING === state
     }
 
-    /** Draft IA with Expired NR */
+    /** Draft Application with Expired NR */
     const isExpired = (item: Business): boolean => {
       return isDraft(status(item)) && (item.nameRequest && (item.nameRequest.expirationDate !== null) &&
-        (new Date(item.nameRequest.expirationDate) < new Date())) && isIA(type(item))
+        (new Date(item.nameRequest.expirationDate) < new Date()))
```

Comment:
Not your code but:

 "parsing dates via the Date object is unreliable and inconsistent across platforms"
"There are different implementations and different behaviors. Sometimes you may get local time. Sometimes you may get UTC. Sometimes you may get Invalid Date. Sometimes you will get different results near DST transitions."

Reply:
better using `moment(item.nameRequest.expirationDate)` or similar

Reply:
What are you thoughts on these?

- https://sumitaec108.medium.com/stop-using-momentjs-f71872f01716
- https://dockyard.com/blog/2020/02/14/you-probably-don-t-need-moment-js-anymore
- https://www.reddit.com/r/javascript/comments/b0ky8z/why_you_shouldnt_use_momentjs/
- https://bootcamp.uxdesign.cc/are-you-still-using-moment-js-in-your-new-project-try-these-four-libraries-instead-8ea84cd50541